### DISCUSSION TITLE
Clarity about VM version on Nextcloud.com

### DISCUSSION THREAD

**Discussion by joshtrichards** [nextcloud/all-in-one#2966] [Questions] [answered]
### Is your feature request related to a problem? Please describe. 

Visiting https://nextcloud.com/install/ under *Download Server* there is not only a link to the **AIO Docker Image** (the one I associate with this repo), but also another to **AIO VM Image** which links to a [random OVA that is over a year old and only includes a four line README](https://download.nextcloud.com/aio-vm/). What is this? How is it built? Is it somehow related to the turnkey VM in https://github.com/nextcloud/vm? (If so, why it is being confusing called AIO too now?)

### Describe the solution you'd like

The Nextcloud.com download links for the various ways of deploying Nextcloud are confusing and vague. Naming should be cleaned up and unified across the NC hosted install projects.

The preferred (recommended) approaches should be highlighted first and each approach listed should include links to their respective source repositories and documentation.

### Additional context

Someone posted on the Help Forum they'd installed the "AIO VM" and followed the "4 step readme" and it didn't work. I was (and still am) confused what they've ended up with. But it seems clear they installed the unclearly sourced and documented VM I linked to above.

https://help.nextcloud.com/t/cant-access-admin-panel-for-the-first-time/166000?u=jtr

**Comment by szaimen** [comment, marked as answer]
Hi, the AIO-VM is simply an OVA that comes with Ubuntu 22.04 LTS and Docker as well as AIO pre-installed.
